napari/_qt/qt_main_window.py

```diff
@@ -326,7 +326,7 @@ def eventFilter(self, source, event):
                         self.menuBar().hide()
             elif event.type() == QEvent.Type.Leave and source is self:
                 self.menuBar().hide()
-        return super().eventFilter(source, event)
+        return False
```

Comment:
why this change?

Reply:
I have been seeing around test failing over PySide6 due to the handling of a `QWidgetItem` event when reaching this line. Errors like https://github.com/napari/napari/actions/runs/11783830702/job/32824335938#step:15:249 and https://github.com/napari/napari/actions/runs/11784483916/job/32824348990?pr=7348#step:15:249

So I was checking is just not calling `super` here could make things pass. However, maybe checking what is going on with that over a separate PR is better?

Reply:
Just in case, with the last commit reverted this change and to explore the error mentioned I opened PR #7367